models/public/colorization-v2-norebal/model.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+    The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+    group of models designed to perform image colorization. For details
+    about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+    The gray-scale or BGR image should be converted to LAB color space first.
+    The model input is a blob that consists of a L-channel of image of 1x1x224x224.
+    The mean values need to be subtracted as follows: 50.0
+    before passing the image blob into the network. In addition, values must be divided
+    by 255.
+
+    The model output is a blob that consists of AB channels of image of 1x2x56x56.
+task_type: "colorization"
+files:
+  - name: colorization-v2-norebal.prototxt
+    sha256: d16418cef8df4ccd703a55ae0ef3960861d5010418f77c90d0a47689998a7169
+    source: https://raw.githubusercontent.com/richzhang/colorization/master/models/colorization_deploy_v2.prototxt
+    size: 9945
+  - name: colorization-v2-norebal.caffemodel
+    size: 128946454
+    sha256: 6ab42a7fa853eb7ca822b9a0a2997a08a29c819c985c3e70b460c6be9c8b2159
+    source: https://people.eecs.berkeley.edu/~rich.zhang/projects/2016_colorization/files/demo_v2/colorization_release_v2_norebal.caffemodel
+  - name: colorization-v2-norebal.npy
```

Comment:
Please explain the purpose of this file.

Reply:
In the original model,  weights of last layer are stored in this file.
So that the model does not change, this file is used.

Reply:
And how this file affects conversion to IR? Or further usage of converted model?